src/main/java/org/example/bookshop/exception/CustomGlobalExceptionHandler.java

```diff
@@ -11,28 +11,42 @@
 import org.springframework.validation.FieldError;
 import org.springframework.validation.ObjectError;
 import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
 import org.springframework.web.context.request.WebRequest;
 import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
 
 @RestControllerAdvice
 public class CustomGlobalExceptionHandler extends ResponseEntityExceptionHandler {
+    private static final Map<String, Object> body = new LinkedHashMap<>();
```

Comment:
Don't use the body as a static variable of class. You can have unexpected problems with responses in the future.